README.md

```diff
@@ -120,25 +125,30 @@ $ git clone https://github.com/softdevteam/krun.git
 $ cd krun
 ```
 
-## Step 3: set `JAVA_HOME`
+## Step 3: Build Krun
+
+The Krun Makefile honours the standard variables: `CC`, `CPPFLAGS`, `CFLAGS`
+and `LDFLAGS`. For example, if you wish to use `clang` rather than `gcc` you
+can append `CC=/bin/clang` to the `make` command here:
```

Comment:
Minor nit-pick. A clang binary is unlikely to appear under `/bin`, on linux it's in `/usr/bin` and on BSD it's under `/usr/local/bin`. To avoid putting any path, you could put something like `CC=/your/path/to/clang` and that would probably suffice.